src/main/java/io/github/rypofalem/armorstandeditor/PlayerEditorManager.java

```diff
@@ -106,7 +108,18 @@ void onArmorStandInteract(PlayerInteractAtEntityEvent event){
 			} else {
 				name = null;
 			}
-
+                        
+                        if (isLocked(as)) {
+                            event.setCancelled(true);
+                            String asName = as.getCustomName();
+                            boolean asNameVisible = as.isCustomNameVisible();
+                            Bukkit.getScheduler().runTaskLater(plugin, () -> {
+                            as.setCustomName(asName);
+                            as.setCustomNameVisible(asNameVisible);
+                            }, 1);
```

Comment:
What is this trying to do? It seems to do nothing, unless I'm missing some Bukkit quirk?